zlibWrapper/Makefile

```diff
@@ -7,8 +7,9 @@
 
 # Paths to static and dynamic zlib and zstd libraries
 ifneq (,$(filter Windows%,$(OS)))
-STATICLIB = ../../zlib/libz.a ../lib/libzstd.a
-IMPLIB    = ../../zlib/libz.dll.a ../lib/libzstd.a
+ZLIBDIR = ../../zlib1.2.8
```

Comment:
I think it's fair to support a limited number of systems.
After all, it's only an example.

Just be clear when a system is not supported, so that it looks like a known limitation rather than an error.